apps/search/js/providers/suggestions.js

```diff
@@ -46,6 +46,10 @@
 
     remote: true,
 
+    clear: function() {
+      this.container.innerHTML = '';
+    },
```

Comment:
Yeh for some reason I was getting unit test failures previous to adding it, but must have been something else, passing now without it